### PR TITLE
initial project files

### PR DESCRIPTION
**Pull request by zaro0508**
Add files for project

* seetup a service account
* update docs
* setup sceptre project files
* setup pre-commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+process.
+
+## Requirements:
+* Install [pre-commit](https://pre-commit.com/#install) app
+* Clone this repo
+* Run `pre-commit install` to install the git hook.
+
+## Testing
+As a pre-deployment step we syntatically validate our sceptre and
+cloudformation yaml files with [pre-commit](https://pre-commit.com).
```
Comment (zxiiro) [inline README.md:16]: For consistancy we should always spell "CloudFormation" and "YAML" imo.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+
+Please install pre-commit, once installed the file validations will
+automatically run on every commit.  Alternatively you can manually
+execute the validations by running `pre-commit run --all-files`.
+
+## Continuous Integration
+We have setup CI to deploy cloudformation template in this project on
```
Comment (zxiiro) [inline README.md:23]: CloudFormation

**Review by zxiiro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zaro0508) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (zaro0508) [reply README.md]: fixed

**Review by zxiiro — APPROVED** [review]

**Pull request merged**